README.md

```diff
@@ -524,6 +525,62 @@ Note: `petName` is only included if the game sent it to your chat via untradeabl
 ```
 </details>
 
+### Player Kills:
+
+`%TARGET%` will be replaced with the victim's user name
```

Comment:
Include note that `world` and `location` will be omitted if the toggle is enabled/disabled
You can also make it an entirely new example - up to but I think both options are clear